spec/std/spec_helper.cr

```diff
@@ -4,3 +4,13 @@ require "../support/tempfile"
 def datapath(*components)
   File.join("spec", "std", "data", *components)
 end
+
+{% if flag?(:win32) %}
+  def pending_win32(description = "assert", file = __FILE__, line = __LINE__, end_line = __END_LINE__, &block)
```

Comment:
This is a pretty straightforward implementation, but having all those `pending_win32` in the specs looks a bit strange. The examples are foremost actual specs and should be called `it`. Maybe `it "foobar", win32: :pending` would be a better alternative?

Reply:
That's overthinking it. Unless you want to make a pull request to `Spec` itself I see no reason to make this look super fancy when all these pendings will be removed sooner or later.

Reply:
Yeah, that's probably not worth it changing `it`. But maybe there is a better name for this method...?

Reply:
Nah, it's clear enough and easy enough to look up. We're hardly writing public API here, code quality and naming of methods in spec files is really not worth bikeshedding about since it's so easy to change.